setup.py

```diff
@@ -1,3 +1,35 @@
-from setuptools import setup
+from setuptools import find_packages, setup
 
-setup()
+extras = {}
+extras["quality"] = [
+    "black ~= 22.10.0",
+    "isort >= 5.5.4",
+    "flake8 >= 3.8.3",
+]
+extras["docs"] = ["sphinx==4.0.0", "sphinx_rtd_theme"]
+extras["test_prod"] = ["pytest", "pytest-xdist", "pytest-subtests", "parameterized"]
```

Comment:
What does this get us over just using `setup.cfg`?

Reply:
setup.cfg is actually the right way. will fix.